src/components/ui/Navigation/Sidebar.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { CSSProperties, ReactElement, useState } from 'react';
+import { NavLink, useRouteMatch } from 'react-router-dom';
+import Sidenav, { Nav } from 'src/components/ui/Navigation/Styles/sidebar';
+import { ReactComponent as KitsuDatabaseTools } from 'src/assets/kitsuDatabaseTools.svg';
+import HamburgerMenu from './HamburgerMenu';
+
+// TODO: make it exitable on very small screens
+// TODO: consider an in-between state using icons for tablets
+export default function Sidebar(): ReactElement {
+  const [open, setOpen] = useState(false);
+
+  let openSesame: CSSProperties;
+  if (open) {
+    openSesame = { '--show-sidebar': 'block' } as CSSProperties;
+  } else {
+    openSesame = { '--show-sidebar': 'none' } as CSSProperties;
+  }
```

Comment:
should be able to refactor this to just be ternery to determine if `block` or `none` and then set `openSesame` to include that.